tests/contrib/BUILD

```diff
@@ -154,3 +154,17 @@ docker_push(
     name = "test_docker_push_empty_bundle",
     bundle = ":create_empty_bundle",
 )
+
+container_bundle(
```

Comment:
move this to https://github.com/bazelbuild/rules_docker/blob/master/tests/docker/BUILD

Reply:
Done